Segment-Mixpanel.podspec

```diff
@@ -21,6 +21,6 @@ Pod::Spec.new do |s|
 
   s.source_files = 'Pod/Classes/**/*'
 
-  s.dependency 'Analytics', '~> 3.0.0'
+  s.dependency 'Analytics', '~> 3'
```

Comment:
@emkman Yes I understand your use case and it should still be 3.0. `'Analytics', '~> 3.0'` means use any version greater than 3.0 and less than 4.0.

Using `'Analytics', '~> 3'` is saying use version 3 and higher, which is the same as not specifying a version. We don't want this because we might make a breaking API change in v4 that will break this pod.

See https://guides.cocoapods.org/syntax/podfile.html#pod and https://guides.cocoapods.org/using/the-podfile.html for reference.

```
'~> 0.1.2' Version 0.1.2 and the versions up to 0.2, not including 0.2 and higher
'~> 0.1' Version 0.1 and the versions up to 1.0, not including 1.0 and higher
'~> 0' Version 0 and higher, this is basically the same as not having it.
```